definitions/npm/graceful-fs_v4.1.x/flow_v0.28.x-/graceful-fs_v4.1.x.js

```diff
@@ -0,0 +1,197 @@
+interface ErrnoError extends Error {
```

Comment:
Please move this into the module declaration.
Otherwise you're creating globals